application/testing/CMakeLists.txt

```diff
@@ -696,6 +696,7 @@ f3d_test(NAME TestInteractionInvertZoom DATA suzanne.ply ARGS --invert-zoom INTE
 f3d_test(NAME TestInteractionCameraHotkeys DATA cow.vtp INTERACTION DEFAULT_LIGHTS)
 f3d_test(NAME TestInteractionZoomToMouse DATA cow.vtp INTERACTION DEFAULT_LIGHTS)
 f3d_test(NAME TestInteractionOrthographicProjection DATA cow.vtp INTERACTION DEFAULT_LIGHTS) #5;5
+f3d_test(NAME TestInteractionOrthographicProjectionCommand ARGS --camera-orthographic DATA cow.vtp INTERACTION DEFAULT_LIGHTS) #Command line option;
```

Comment:
nope, its not an interaction test, just a standard test

Reply:
Offcourse!